packages/node_modules/@ciscospark/react-container-message-composer/src/components/ComposerButtons.test.js

```diff
@@ -10,6 +10,9 @@ describe('ComposerButtons component', () => {
 
   beforeEach(() => {
     props = {
+      composerActions: {
```

Comment:
can you add another test with attachFiles being false?

Reply:
yeah i will. good catch, thanks!!